style.css

```diff
@@ -137,3 +137,176 @@ nav {
     opacity: 0.5;
   }
 }
+
+.recent-works {
+  margin: 0 7vw;
+}
+
+.recent-works-header {
+  margin-bottom: 4rem;
+}
+
+.recent-works-line {
+  width: 15vw;
+  height: 1vh;
+  background-color: #36b37e;
+  border-radius: 1.5rem;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: repeat(6, auto);
+  gap: 1rem;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 52% 48%;
+  background-color: #d0d9d4;
+  border-radius: 10px;
+  height: 90vh;
+}
+
+.card-image {
+  height: 16rem;
+}
+
+.card-text {
+  background-color: #ebf0ee;
+  text-align: center;
+  border-radius: 10px;
+  border: #d0d9d4 5px;
+  margin: 1px;
+}
+
+.card-title {
+  color: #3a4a42;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  padding: 0 3rem;
+}
+
+.card-tools-list {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.card-tool {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  background-color: #d0d9d4;
+  padding: 8px 12px;
+  border-radius: 4px;
+}
+
+.see-project {
+  background: #36b37f;
+  border-radius: 4px;
+  border: none;
+  width: 122px;
+  height: 40px;
+  color: #fff;
+  letter-spacing: 0.03em;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.about-me {
+  background-image: url(./images/mobile/illustration-\ about\ me-\ mobile.png);
+  background-repeat: no-repeat;
+  margin-bottom: 5rem;
+}
```

Comment:
- [ ] It is important that you wrap the contents of the about me section using flexbox as this would come in very handy in further milestones. please also endeavour to go with conventions designed to make you build cleaner and responsive layouts, css grids and flexbox are ones of these :+1: